Sapporo/Sources/Section/SASectionIndexType.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  SASectionIndexType.swift
+//  Example
+//
+//  Created by 川上 健太郎 on 2018/08/13.
+//  Copyright © 2018年 Le Van Nghia. All rights reserved.
+//
+
+import Foundation
+
+public protocol SASectionIndexType {
+    var intValue: Int { get }
+    static var count: Int { get }
+}
```

Comment:
It may makes more sense like this:
```swift
public protocol SASectionIndexType: RawRepresentable where Self.RawValue == Int {
    static var count: Int { get }
}
```
`intValue` is no longer needed.
And, In Swift4.2, we can use CaseIterable in future.